lib/http/http.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+require "net/http"
+require "json"
+
+module Http
+  # An http client
+  class Client
+    def call_post(url, req_body, headers)
+      http = Net::HTTP.new(url.host, url.port)
+      http.use_ssl = true
```

Comment:
what if url is not https? I see you support to pass any url into this

Reply:
I was added the optional `disable_ssl` in case user want to disable ssl (almost host use https this time so I want to default enable it)